doc/modules/changes/20171001_mbweller

```diff
@@ -0,0 +1,5 @@
+New: The 'function' boundary composition plugin now allows user defined
+Coordinate Systems [cartesian; spherical; depth].
```

Comment:
Since that 'function' plugin didn't previously exist, I think this is misleading.